### PR TITLE
fix: cleanup status colors

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Cleanup some more old colors :) 
- Slowly getting there

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### Preview domain
https://fix-cleanup-colors.preview.app.daily.dev

### CODE REVIEW

File: packages/extension/src/newtab/ShortcutLinksModal.tsx [hunk 1]
```diff
@@ -85,7 +86,8 @@ export default function CustomLinksModal({
           <Button
             onClick={onRevokePermission}
             form="shortcuts-modal"
-            variant={ButtonVariant.Secondary}
+            variant={ButtonVariant.Primary}
+            color={ButtonColor.Ketchup}
```
Comment (nensidosari) [inline packages/extension/src/newtab/ShortcutLinksModal.tsx:90]: Is this by mistake or a requested fix?

**Review by nensidosari — APPROVED** [review]
Looks great

**Review by capJavert — APPROVED** [review]

File: packages/extension/src/newtab/ShortcutLinksModal.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/extension/src/newtab/ShortcutLinksModal.tsx:90]: Toast fix i apparently didnt push 🙈

**Review by sshanzel — APPROVED** [review]
Looks great! Lots of cleanups.

**Pull request merged**